content/en/docs/how-tos/cluster-claim.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Creating a Cluster Pool"
+description: How to create a cluster pool from which a job can claim a cluster.
+---
+
+DPTP integrates Hive into the OpenShift test platform to manage the cluster pools and clusters in them. There is a defined pool [ci-cluster-pool](https://github.com/openshift/release/tree/master/clusters/app.ci/ci-cluster-pool) from which can be used to claim an OpenShift cluster for a test. In this page, we document how to create a new cluster pool with the cloud account provided by a CI user.
+
+## Prerequisites
+
+*  Choose a [Hive](https://github.com/openshift/hive#supported-cloud-providers) supported cloud for the pool.
```

Comment:
```suggestion
* Choose a [Hive](https://github.com/openshift/hive#supported-cloud-providers) supported cloud for the pool.
```

Reply:
And probably more something like "Vefirify that the cloud platform you want to use is supported by Hive".